agent/bench-scripts/templates/fio-shared-fs.job

```diff
@@ -0,0 +1,35 @@
+# Syntax:
+#   $target : expands to the fio target file
```

Comment:
Should this read, `expands to the fio target directory`?

Reply:
yes

Reply:
done in latest commit.